src/main/deploy/auton/RedAmpWing1Center1.xml

```diff
@@ -2,25 +2,32 @@
 <!DOCTYPE auton SYSTEM "auton.dtd">
 <auton>
 
-<primitive id = "RESET_POSITION_PATH_PLANNER"
-            pathname= "BlueAmpWing1Center1"
-                noteOption="STATE_HOLD_FEEDER"/>
+<primitive id= "RESET_POSITION_PATH_PLANNER_NO_VISION"
```

Comment:
Same comment as above

Reply:
Again AMP side and I thought this was like this before, so I'm surprised it is new